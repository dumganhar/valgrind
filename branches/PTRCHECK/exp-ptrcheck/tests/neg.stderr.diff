12,15d11
< 
< Invalid read of size 4
<    at 0x........: main (neg.c:16)
<  Address 0x........ is not derived from any known block
17,20d12
< 
< Invalid read of size 4
<    at 0x........: main (neg.c:16)
<  Address 0x........ is not derived from any known block
22,25d13
< 
< Invalid read of size 4
<    at 0x........: main (neg.c:18)
<  Address 0x........ is not derived from any known block
27,30d14
< 
< Invalid read of size 4
<    at 0x........: main (neg.c:18)
<  Address 0x........ is not derived from any known block
52c36
< ERROR SUMMARY: 10 errors from 10 contexts (suppressed: 0 from 0)
---
> ERROR SUMMARY: 6 errors from 6 contexts (suppressed: 0 from 0)
