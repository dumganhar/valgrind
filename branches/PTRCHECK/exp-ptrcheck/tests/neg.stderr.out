
about to do 14 [0]
Invalid read of size 4
   at 0x........: main (neg.c:14)
 Address 0x........ is not derived from any known block
about to do 14 [-1]

Invalid read of size 4
   at 0x........: main (neg.c:14)
 Address 0x........ is not derived from any known block
about to do 16 [0]
about to do 16 [-1]
about to do 18 [0]
about to do 18 [-1]
about to do 20 [0]

Invalid read of size 4
   at 0x........: main (neg.c:20)
 Address 0x........ is not derived from any known block
about to do 20 [-1]

Invalid read of size 4
   at 0x........: main (neg.c:20)
 Address 0x........ is not derived from any known block
about to do 22 [0]

Invalid read of size 4
   at 0x........: main (neg.c:22)
 Address 0x........ is not derived from any known block
about to do 22 [-1]

Invalid read of size 4
   at 0x........: main (neg.c:22)
 Address 0x........ is not derived from any known block

ERROR SUMMARY: 6 errors from 6 contexts (suppressed: 0 from 0)
